src/plugins/ui/sheetview.ts

```diff
@@ -156,8 +156,12 @@ export class SheetViewPlugin extends UIPlugin {
   }
 
   handle(cmd: Command) {
-    this.cleanViewports();
-
+    switch (cmd.type) {
+      case "UNDO":
+      case "REDO":
```

Comment:
undo/redo are already handled (differently) below ? is it normal ?

Reply:
the cleanViewport is a pre-step, unrelated to the actual action we want to do with undo/redo but I could add 'cleanViewports' in  the al-ready existing switch case. It seemed clearer in my head to isolate this logic